includes/admin/events/metaboxes.php

```diff
@@ -609,6 +609,7 @@ function mdjm_event_metabox_options_playlist_row( $event_id )	{
 	global $mdjm_event, $mdjm_event_update;
 
 	$enable_playlist = mdjm_get_option( 'enable_playlists', true );
+	$pl_limit = $mdjm_event->get_meta( '_mdjm_event_playlist_limit' );
```

Comment:
Please use `$playlist_limit` rather than `$pl_limit`. Also align the variables... i.e.

    $enable_playlist = mdjm_get_option( 'enable_playlists', true );
    $playlist_limit  = $mdjm_event->get_meta( '_mdjm_event_playlist_limit' );